tests/src/python/test_provider_ogr.py

```diff
@@ -1092,7 +1092,10 @@ def testBoolFieldEvaluation(self):
         self.assertEqual(vl.fields().at(0).name(), 'bool')
         self.assertEqual(vl.fields().at(0).type(), QVariant.Bool)
         self.assertEqual([f[0] for f in vl.getFeatures()], [True, False, NULL])
-        self.assertEqual([f[0].__class__.__name__ for f in vl.getFeatures()], ['bool', 'bool', 'QVariant'])
```

Comment:
Do you think we could safely drop this test? I think it's already covered anyway by line 1094

Reply:
agreed. I've just dropped it